tests-semanticdb/src/test/scala/scala/meta/tests/semanticdb/TargetedSuite.scala

```diff
@@ -238,6 +238,61 @@ class TargetedSuite extends SemanticdbSuite {
     }
   )
 
+  test("named-args") {
+    val code = """|@deprecated(since = "123")
+                  |class CLS(name: String) {
+                  |  def this(name2: Int) = this(name = name2.toString)
+                  |}
+                  |""".stripMargin
+    assertEquals(
+      computePayloadFromSnippet(code).linesIterator.drop(3).filter(!_.startsWith("Uri => "))
+        .mkString("", "\n", "\n"),
+      """|Summary:
+         |Schema => SemanticDB v4
+         |Text => non-empty
+         |Language => Scala
+         |Symbols => 8 entries
+         |Occurrences => 14 entries
+         |
+         |Symbols:
+         |_empty_/CLS# => @deprecated class CLS extends AnyRef { +3 decls }
+         |  deprecated => scala/deprecated#
+         |  AnyRef => scala/AnyRef#
+         |_empty_/CLS#`<init>`(). => primary ctor <init>(name: String)
+         |  name => _empty_/CLS#`<init>`().(name)
+         |  String => scala/Predef.String#
+         |_empty_/CLS#`<init>`().(name) => param name: String
+         |  String => scala/Predef.String#
+         |_empty_/CLS#`<init>`(+1). => ctor <init>(name2: Int)
+         |  name2 => _empty_/CLS#`<init>`(+1).(name2)
+         |  Int => scala/Int#
+         |_empty_/CLS#`<init>`(+1).(name2) => param name2: Int
+         |  Int => scala/Int#
+         |_empty_/CLS#name. => private[this] val method name: String
+         |  String => scala/Predef.String#
+         |local0 => val local x$1: "123"
+         |local1 => val local x$2: String
+         |  String => scala/Predef.String#
+         |
+         |Occurrences:
+         |[0:1..0:11): deprecated => _empty_/
```

Comment:
why is this empty? is that expected?

Reply:
Hmm... not really, but looks like an older bug, so maybe I will raise a separate issue? Seems to only happen when no package is defined

Reply:
btw, looks like in 2.11 and 2.12 it's not empty but i don't know about 2.13.15... that's why i like to add tests first, to establish baseline, before making changes.

Reply:
Uch, looks like that is broken also :/ This is a very unlucky release

Reply:
`2.13.15` is the same. Will fix it separately

Reply:
https://github.com/scalameta/scalameta/issues/4136